src/components/Navbar/Navbar.tsx

```diff
@@ -47,6 +49,7 @@ export default function Navbar() {
         </button>
       </div>
       <input id='search-box' placeholder={`Search for ${searchContent}`} value={search} onChange={onSearchChange} />
+      <i className='fa fa-sign-out' onClick={handleLogout}></i>
```

Comment:
Accessibility has not been taken into consideration. Please create a new placeholder ticket that could be revisited in the future. Set it for now as a low priority.

Reply:
Please, can you check is it good now?